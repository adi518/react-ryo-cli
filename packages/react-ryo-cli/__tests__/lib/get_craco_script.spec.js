const { SCRIPTS } = require("../../lib/constants");
const getCracoScript = require("../../lib/helpers/get_craco_script");

describe("getCracoScript", () => {
  test("build", () => {
    expect(getCracoScript(SCRIPTS.BUILD)).toMatchSnapshot();
  });

  test("build", () => {
    expect(getCracoScript(SCRIPTS.BUILD)).toMatchSnapshot();
  });

  test("build:development", () => {
    expect(getCracoScript(SCRIPTS.BUILD_DEVELOPMENT)).toMatchSnapshot();
  });

  test("build:package", () => {
    expect(getCracoScript(SCRIPTS.BUILD_PACKAGE)).toMatchSnapshot();
  });

  test("build:package:development", () => {
    expect(getCracoScript(SCRIPTS.BUILD_PACKAGE_DEVELOPMENT)).toMatchSnapshot();
  });

  test("build:stats", () => {
    expect(getCracoScript(SCRIPTS.BUILD_STATS)).toMatchSnapshot();
  });

  test("test", () => {
    expect(getCracoScript(SCRIPTS.TEST)).toMatchSnapshot();
  });

  test("test:watch", () => {
    expect(getCracoScript(SCRIPTS.TEST_WATCH)).toMatchSnapshot();
  });

  test("test:update", () => {
    expect(getCracoScript(SCRIPTS.TEST_UPDATE)).toMatchSnapshot();
  });

  test("test:production", () => {
    expect(getCracoScript(SCRIPTS.TEST_PRODUCTION)).toMatchSnapshot();
  });

  test("eject", () => {
    expect(getCracoScript(SCRIPTS.EJECT)).toMatchSnapshot();
  });
});
