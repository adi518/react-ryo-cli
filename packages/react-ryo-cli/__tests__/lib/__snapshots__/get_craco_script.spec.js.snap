// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCracoScript build 1`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\@craco\\\\craco\\\\bin\\\\craco.js",
  "build",
  "--config",
  "D:\\\\dev\\\\@react-ryo-cli\\\\packages\\\\react-ryo-cli\\\\config\\\\craco\\\\craco.config.js",
]
`;

exports[`getCracoScript build 2`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\@craco\\\\craco\\\\bin\\\\craco.js",
  "build",
  "--config",
  "D:\\\\dev\\\\@react-ryo-cli\\\\packages\\\\react-ryo-cli\\\\config\\\\craco\\\\craco.config.js",
]
`;

exports[`getCracoScript build:development 1`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\@craco\\\\craco\\\\bin\\\\craco.js",
  "build",
  "--config",
  "D:\\\\dev\\\\@react-ryo-cli\\\\packages\\\\react-ryo-cli\\\\config\\\\craco\\\\craco.config.js",
]
`;

exports[`getCracoScript build:package 1`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\@craco\\\\craco\\\\bin\\\\craco.js",
  "build",
  "--config",
  "D:\\\\dev\\\\@react-ryo-cli\\\\packages\\\\react-ryo-cli\\\\config\\\\craco\\\\craco.config.js",
]
`;

exports[`getCracoScript build:package:development 1`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\@craco\\\\craco\\\\bin\\\\craco.js",
  "build:package:development",
  "--config",
  "D:\\\\dev\\\\@react-ryo-cli\\\\packages\\\\react-ryo-cli\\\\config\\\\craco\\\\craco.config.js",
]
`;

exports[`getCracoScript build:stats 1`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\source-map-explorer\\\\dist\\\\cli.js",
  "undefined/static/js/*.js",
]
`;

exports[`getCracoScript eject 1`] = `
Array [
  "eject",
]
`;

exports[`getCracoScript test 1`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\@craco\\\\craco\\\\bin\\\\craco.js",
  "test",
  "--config",
  "D:\\\\dev\\\\@react-ryo-cli\\\\packages\\\\react-ryo-cli\\\\config\\\\craco\\\\craco.config.js",
]
`;

exports[`getCracoScript test:production 1`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\@craco\\\\craco\\\\bin\\\\craco.js",
  "test",
  "--ci",
  "--collectCoverage",
  "--config",
  "D:\\\\dev\\\\@react-ryo-cli\\\\packages\\\\react-ryo-cli\\\\config\\\\craco\\\\craco.config.js",
]
`;

exports[`getCracoScript test:update 1`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\@craco\\\\craco\\\\bin\\\\craco.js",
  "test",
  "--updateSnapshot",
  "--config",
  "D:\\\\dev\\\\@react-ryo-cli\\\\packages\\\\react-ryo-cli\\\\config\\\\craco\\\\craco.config.js",
]
`;

exports[`getCracoScript test:watch 1`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\@craco\\\\craco\\\\bin\\\\craco.js",
  "test",
  "--config",
  "D:\\\\dev\\\\@react-ryo-cli\\\\packages\\\\react-ryo-cli\\\\config\\\\craco\\\\craco.config.js",
]
`;
